### PR TITLE
SDK-682: update manage product API

### PR DESCRIPTION
**Pull request by hailefrollo**

### CODE REVIEW

File: Sources/FrolloSDK/Model/Transient/ManagedProduct.swift [hunk 1]
```diff
@@ -74,4 +78,24 @@ public class ManagedProduct: Codable {
         
     }
     
+    public class Feature: Codable {
+        /// id of Feature
+        public var id: Int64
+        
+        /// description of `Feature`
+        public var description: String
```
Comment (sincychacko-frollo) [inline Sources/FrolloSDK/Model/Transient/ManagedProduct.swift:86]: Can you please confirm if all these fields are required?

**Review by sincychacko-frollo — APPROVED** [review]

File: Sources/FrolloSDK/Model/Transient/ManagedProduct.swift (hunk 1 above)
Comment (hailefrollo) [reply Sources/FrolloSDK/Model/Transient/ManagedProduct.swift:86]: sure

File: Sources/FrolloSDK/Model/Transient/ManagedProduct.swift [hunk 2]
```diff
@@ -74,4 +78,24 @@ public class ManagedProduct: Codable {
         
     }
     
+    public class Feature: Codable {
```
Comment (dipeshfrollo) [inline Sources/FrolloSDK/Model/Transient/ManagedProduct.swift:81]: Documentation missing

**Review by dipeshfrollo — APPROVED** [review]

File: Sources/FrolloSDK/Model/Transient/ManagedProduct.swift (hunk 1 above)
Comment (hailefrollo) [reply Sources/FrolloSDK/Model/Transient/ManagedProduct.swift:91]: Payam confirmed those fields are required. :)

**Pull request merged**